ci/builders/linux_unopt.json

```diff
@@ -0,0 +1,122 @@
+                    "name": "test: Host_Tests_for_host_debug_unopt",
+                    "parameters": [
+                        "--variant",
+                        "host_debug_unopt",
+                        "--type",
+                        "dart,engine",
+                        "--engine-capture-core-dump",
+                        "--use-sanitizer-suppressions"
+                    ],
+                    "script": "flutter/testing/run_tests.py"
+                },
+                {
+                    "name": "analyze_dart_ui",
+                    "script": "flutter/ci/analyze.sh"
+                },
+                {
+                    "language": "dart",
+                    "name": "test: observatory and service protocol",
+                    "script": "flutter/shell/testing/observatory/test.dart",
+                    "parameters": [
+                        "out/host_debug_unopt/flutter_tester",
+                        "flutter/shell/testing/observatory/empty_main.dart"
+                    ]
+                }
+            ]
+        },
+        {
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux"
+            ],
+            "dependencies": [
+                {
+                    "dependency": "arm_tools",
+                    "version": "last_updated:2023-02-03T15:32:01-0800"
+                }
+            ],
+            "gn": [
+                "--android",
+                "--unoptimized"
+            ],
+            "name": "android_debug_unopt",
+            "ninja": {
+                "config": "android_debug_unopt",
+                "targets": [
+                    "flutter/impeller",
+                    "flutter/shell/platform/android:android_jar",
+                    "flutter/shell/platform/android:robolectric_tests"
+                ]
+            },
+            "tests": [
+                {
+                    "language": "python3",
+                    "name": "test: Host Tests for android_debug_unopt",
+                    "parameters": [
+                        "--variant",
+                        "android_debug_unopt",
+                        "--type",
+                        "java",
+                        "--engine-capture-core-dump",
+                        "--android-variant",
+                        "android_debug_unopt"
+                    ],
+                    "script": "flutter/testing/run_tests.py"
+                },
+                {
+                    "language": "python3",
+                    "name": "malioc diff",
+                    "parameters": [
+                        "--before-relative-to-src",
+                        "flutter/impeller/tools/malioc.json",
+                        "--after-relative-to-src",
+                        "out/android_debug_unopt/gen/malioc",
+                        "--print-diff"
+                    ],
+                    "script": "flutter/impeller/tools/malioc_diff.py"
```

Comment:
This won't do anything unless the path to malioc is passed to `gn` like here https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/engine/engine_unopt.py#127.

Reply:
The arm-tool dep is setting up the env variable that gn looks for: https://cs.opensource.google/flutter/recipes/+/main:recipe_modules/flutter_deps/api.py;l=173

Reply:
This is a led run with the new configs: https://luci-milo.appspot.com/raw/build/logs.chromium.org/flutter/led/godofredoc_google.com/d084962a4229fdc42c98670a8e9f95c1c8d397dac0deb4876b58714a0b8d21a4/+/build.proto